### PR TITLE
feat(gitea): split gitea into two controller deployment

### PR DESCRIPTION
**Pull request by cboin1996**
part of: https://github.com/StatCan/daaas/issues/890

### CODE REVIEW

**Review by wg102 — DISMISSED** [review]
One comment for all

File: stable/profiles-controller/templates/deployment-gitea-protected-b.yaml [hunk 1]
```diff
@@ -36,15 +36,17 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
             - gitea
-            - --argocdnamespace 
-            - {{ .Values.components.gitea.namespace }}
           envFrom:
-              {{- toYaml .Values.components.gitea.envFrom | nindent 14 }}
+              {{- toYaml .Values.components.giteaProtectedb.envFrom | nindent 14 }}
```
[the remark this answers is not in the document]
Comment (cboin1996) [reply stable/profiles-controller/templates/deployment-gitea-protected-b.yaml]: Addressed in: https://github.com/StatCan/charts/pull/254/commits/42252b9c85b6dd04dd16b7f4c40183f02f6d23bc

File: stable/profiles-controller/templates/deployment-gitea-protected-b.yaml [hunk 2]
```diff
@@ -36,15 +36,17 @@ spec:
...
           env:
             {{- range $env := .Values.extraEnv }}
             - name: {{ $env.name }}
               value: "{{ $env.value }}"
             {{- end }}
+            {{- range $env := .Values.components.giteaProtectedb.env }}
```
[the remark this answers is not in the document]
Comment (cboin1996) [reply stable/profiles-controller/templates/deployment-gitea-protected-b.yaml]: Addressed in: https://github.com/StatCan/charts/pull/254/commits/42252b9c85b6dd04dd16b7f4c40183f02f6d23bc

File: stable/profiles-controller/values.yaml [hunk 3]
```diff
@@ -95,12 +95,61 @@ components:
...
+    - name: GITEA_ARGOCD_SOURCE_TARGET_REVISION
+      value: "aaw-dev-cc-00"
+    - name: GITEA_ARGOCD_SOURCE_PATH
+      value: "profiles-argocd-system/template/gitea/unclassified"
+    - name: GITEA_ARGOCD_PROJECT
+      value: "default"
+    - name: GITEA_SOURCE_CONTROL_ENABLED_LABEL
+      value: "sourcecontrol.statcan.gc.ca/enabled"
+
+  giteaProtectedb:
```
[the remark this answers is not in the document]
Comment (cboin1996) [reply stable/profiles-controller/values.yaml]: Addressed in: https://github.com/StatCan/charts/pull/254/commits/42252b9c85b6dd04dd16b7f4c40183f02f6d23bc

**Review by wg102 — APPROVED** [review]

**Pull request merged**